src/verifiers/openAttestationEthereumTokenRegistryMinted.test.ts

```diff
@@ -43,6 +44,92 @@ describe("openAttestationEthereumTokenRegistryMinted", () => {
     });
   });
   describe("v2", () => {
+    it("should return an invalid fragment when token registry is invalid", async () => {
+      const fragment = await openAttestationEthereumTokenRegistryMinted.verify(
+        {
+          ...documentRopstenNotIssuedWithTokenRegistry,
+          data: {
+            ...documentRopstenNotIssuedWithTokenRegistry.data,
+            issuers: [
+              {
+                ...documentRopstenNotIssuedWithTokenRegistry.data.issuers[0],
+                tokenRegistry: "0fb5b63a-aaa5-4e6e-a6f4-391c0f6ba423:string:0xabcd"
+              }
+            ]
+          }
+        },
+        {
+          network: "ropsten"
+        }
+      );
+      expect(fragment).toStrictEqual({
+        name: "OpenAttestationEthereumTokenRegistryMinted",
+        type: "DOCUMENT_STATUS",
+        data: {
+          details: [
+            {
+              address: "0xabcd",
+              minted: false,
+              reason: {
+                code: 2,
+                codeString: "CONTRACT_ADDRESS_INVALID",
+                message: "Contract address 0xabcd is invalid"
+              }
+            }
+          ],
+          mintedOnAll: false
```

Comment:
i don't know if this matters or not, but business rules imply its impossible to have more than one tokenRegistry - i think we were supposed to work it into the schema at some point but i dont think its been done yet

Reply:
I think the verifier fail if there are 2 tokens, will make sure of this :)

Reply:
there is a check to allow only one token, I just followed the other verifier structure